clippy_dev/src/lib.rs

```diff
@@ -31,9 +30,10 @@ lazy_static! {
     )
     .unwrap();
     static ref NL_ESCAPE_RE: Regex = Regex::new(r#"\\\n\s*"#).unwrap();
-    pub static ref DOCS_LINK: String = "https://rust-lang.github.io/rust-clippy/master/index.html".to_string();
 }
 
+pub static DOCS_LINK: &str = "https://rust-lang.github.io/rust-clippy/master/index.html";
```

Comment:
Why not `const`?

Reply:
I had an debate with people on rustup. It seems that people prefer `static str` rather than 
`const str`. They like that because static item have static address in binary.